tools/composer.json

```diff
@@ -7,5 +7,10 @@
         "phpstan/phpstan": "^1.0.0",
         "infection/infection": "^0.26.1",
         "friendsofphp/php-cs-fixer": "^3.1"
+    },
+    "config": {
+        "allow-plugins": {
+            "infection/extension-installer": true
```

Comment:
Do we need this to be set to true?

Reply:
I'm unsure. My thinking, potentially naïvely, was that this would be one of those things that caused CI to fall over.

Reply:
Yep, it was my naïvety. :innocent: 

Running `composer install --no-interaction` doesn't stop to request permission, so I have dropped that change from the commit!